core/src/main/java/ch/cyberduck/core/SessionPoolFactory.java

```diff
@@ -97,9 +97,9 @@ public static SessionPool create(final ConnectionService connect, final Transcri
                     log.info(String.format("Create new pooled connection pool for %s", bookmark));
                 }
                 return new DefaultSessionPool(connect, trust, key, registry, transcript, bookmark)
-                        .withMinIdle(PreferencesFactory.get().getInteger("connection.pool.minidle"))
-                        .withMaxIdle(PreferencesFactory.get().getInteger("connection.pool.maxidle"))
-                        .withMaxTotal(PreferencesFactory.get().getInteger("connection.pool.maxtotal"));
+                        .withMinIdle(new HostPreferences(bookmark).getInteger("connection.pool.minidle"))
```

Comment:
Any reason to create three instances of HostPreferences? Can't we use one instance for all three properties?